app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        self.alive.append(self)
+        self.dead_remove()
+
+    def dead_remove(self) -> None:
+        for animal in self.alive:
+            if animal.health < 1:
+                self.alive.remove(animal)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> None:
+        if self.hidden:
+            self.hidden = False
+        else:
+            self.hidden = True
```

Comment:
you can write this in one line

Reply:
I created another branch with pull request -https://github.com/mate-academy/py-herbivores-and-carnivores/pull/248. Had problems with pushing in Laptop